scripts/win32.bat

```diff
@@ -5,12 +5,27 @@ exit /b
 
 ----- Begin wsf script --->
 <job><script language="VBScript">
+
+Private OSDrive
+
 strComputer = "."
 
 Set objWMIService = GetObject("winmgmts:\\" & strComputer & "\root\cimv2")
 
 Set colDiskDrives = objWMIService.ExecQuery("SELECT * FROM Win32_DiskDrive")
 
+Set colOperatingSystems = objWMIService.ExecQuery ("SELECT SystemDrive FROM Win32_OperatingSystem")
+
+For Each objOperatingSystem in colOperatingSystems
+    On Error Resume Next
+    '' get the OS System Drive if exists
+    OSDrive = objOperatingSystem.Properties_("SystemDrive")   
+    If Err.Number <> 0 Then
+        OSDrive = Err.Description
```

Comment:
Correct..

Originally I had `OSDrive = False`, but during my various tests, somehow reverted back to `Err.Description`.

The original intention behind `Err.Description` was to provide a meaningful error to `system: {value}` because there'd be no other way to debug the case where `SystemDrive` was not set as a property. However that's probably over-engineering.

I agree that `OSDrive = False` is probably sufficient enough.